File: 130/exercises/easy_2/2.rb
def zip(arr1, arr2)
  new_arr = []
  arr1.size.times do |idx|
    new_arr << [arr1[idx], arr2[idx]]
  end
  new_arr
end

p zip([1, 2, 3], [4, 5, 6]) == [[1, 4], [2, 5], [3, 6]]

File: 210/exercises/small_problems/easy_4/2.js
function union(arr1, arr2) {
  let result = [];
  let i;
  for (i = 0; i < arr1.length; i += 1) {
    if (!result.includes(arr1[i])) {
      result.push(arr1[i]); 
    }
  } 
  for (i = 0; i < arr2.length; i += 1) {
    if (!result.includes(arr2[i])) {
      result.push(arr2[i]); 
    }
  }
  return result;
}

console.log(union([1, 3, 5], [3, 6, 9]));    // [1, 3, 5, 6, 9]
